frontends/main/src/page-components/LearningResourceDrawer/LearningResourceDrawer.tsx

```diff
@@ -142,13 +142,20 @@ const getOpenDrawerSearchParams = (
   return newSearchParams
 }
 
+// TODO Not used anywhere in project. Remove?
```

Comment:
You could remove it. This dates from when we opened the drawer via a click handler, as opposed to the whole card being a link.